pkg/webhook/webhook.go

```diff
@@ -621,6 +629,51 @@ func getResourceList(resourceRequests map[string]int64) *corev1.ResourceList {
 	return &resourceList
 }
 
+func createCustomizedPatch(pod corev1.Pod) ([]jsonPatchOperation, error) {
+	var customizedPatch []jsonPatchOperation
+
+	// lock for reading
+	cusInjects.Lock()
+	defer cusInjects.Unlock()
+
+	for k, v := range cusInjects.Patchs {
+		// The cusInjects will be injected when:
+		// 1. Pod labels contain the patch key defined in cusInjects, and
+		// 2. The value of patch key in pod labels(not in cusInjects) is "true"
+		if podValue, exists := pod.ObjectMeta.Labels[k]; exists && podValue == "true" {
```

Comment:
do you think its worthwhile to cast podValue to lowercase characters before comparison with true?

Reply:
Yes, it'd be better to use lowercase.
Updated.